build/rmutil/Makefile

```diff
@@ -47,11 +59,11 @@ clean:
 
 $(BINDIR)/%.o: $(SRCDIR)/%.c
 	@echo Compiling $<...
-	$(SHOW)$(CC) $(CC_FLAGS) -c $< -o $@
```

Comment:
Maybe just `SHOW ?= 1` on top of Makefile?

Reply:
thank you for the fast review @rafie :+1: